lib/spack/spack/cmd/concretize.py

```diff
@@ -14,11 +14,26 @@ def setup_parser(subparser):
     subparser.add_argument(
         '-f', '--force', action='store_true',
         help="Re-concretize even if already concretized.")
+    subparser.add_argument(
+        '--test', default=None,
+        choices=['root', 'all'],
+        help="""If 'root' is chosen, run package tests during
+installation for top-level packages (but skip tests for dependencies).
+if 'all' is chosen, run package tests during installation for all
+packages. If neither are chosen, don't run tests for any packages.""")
```

Comment:
This isn't entirely accurate. It concretizes including test dependencies, but it doesn't ensure that the tests are actually run.

Reply:
Yeah, let me drop that, this is copy-pasta from the `install` command of course